index.html

```diff
@@ -22,197 +22,223 @@
           </a>
           <nav class="menu">
             <img src="./images/Union.png" alt="Menu">          
-          </nav>          
+          </nav>
+          <!-- <div class="nav-item menu-desktop"> -->
```

Comment:
- [ ] Please consider changing this image with your logo name or just your name text, in any case, it should be something proper to you, using `my logo` as the logo is much  I can say, and not professional at all.

<img width="361" alt="3" src="https://user-images.githubusercontent.com/77843050/208068766-b00fc47d-a3e5-44c3-8344-8ed29eef2b58.png">

<img width="392" alt="4" src="https://user-images.githubusercontent.com/77843050/208068771-65133286-b47e-473a-9ecc-ce162da15002.png">

Reply:
Hi @joskalenda, Thanks for the feedback. I have changed the Logo with my nickname.